### PR TITLE
Reverts Emote System

### PR DESCRIPTION
**Pull request by Fox-McCloud**
After having discussed this over with @TheDZD  over a few days and (in addition to some other coders), it's been decided to revert the system.

As it currently stands, it has too many bugs, some major major performance issues (that can easily result in server crashes), and has a number of useability issues that make it unviable, at the moment.

We're not opposed to re-visiting this system, once it's a bit more well-rounded out and tested, but in its current state, it's not ready for the live server, and it's not something we can justify yet (especially with the primary coder who's handling it being super busy with real life stuffs ATM).

Hopefully we can get this back though, some time!

Tested and everything seems to work ok.

:cl: Fox McCloud
remove: Reverts New emote system; Old emote system is back
/:cl:

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
you mean I won't have the emote tab anymore?

THANK GOD I WAS GETTING SO MANY TABS *scream

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -23,7 +23,6 @@
 	else
 		living_mob_list += src
 	prepare_huds()
-	emoteHandler = new /datum/emoteHandler(src)
```
Comment (Fox-McCloud) [inline code/modules/mob/mob.dm]: This right here, is the reason why server crashes are so easy to produce.

**Comment by VampyrBytes** [comment]
Thanks, that takes the pressure off.

I'll chip away at it when I get the time. One day I will get rid of the abomination that is the current system :wink:

**Comment by Spacemanspark** [comment]
Oh, before I forget.
Can you leave in the ability to emote in dchat?

**Comment by Fox-McCloud** [comment]
>Can you leave in the ability to emote in dchat?

In another PR; this isn't a feature PR so much as getting things back to the way they were.

If someone wants to make a PR after the revert that does just that, I'm definitely not opposed to it, but it's out of the scope of this PR.

**Comment by KasparoVy** [comment]
Just took in how flipping has no cooldown, hahaha. If you're fast enough on the macro and enter key you can keep yourself upside-down indefinitely. Jokes.

**Pull request merged**